src/components/Footer.tsx

```diff
@@ -46,7 +47,7 @@ const Footer = () => {
                 href="https://vercel.com/?utm_source=octoclairvoyant-team&utm_campaign=oss"
                 isExternal
               >
-                <Image src="/powered-by-vercel.svg" width={212} height={44} />
+                <Image src={poweredByVercelLogo} />
```

Comment:
Does this still display at the proper size? Are we sure we're not stretching/squashing the logo?

Reply:
It still displays properly on the preview.

Reply:
Sorry, [I should have link corresponding migration guide for it](https://nextjs.org/blog/next-11#automatic-size-detection-local-images).

With Next.js v11 you can import the images so 1) Next.js knows the dimensions directly and 2) it can generate the blurred version to be displayed while loading.